rfc7991bis.rnc

```diff
@@ -28,7 +28,7 @@ rfc =
    attribute iprExtract { xsd:IDREF }?,
    [ a:defaultValue = "IETF" ]
    attribute submissionType {
-     "IETF" | "IAB" | "IRTF" | "independent"
+     "IETF" | "IAB" | "IRTF" | "independent" | "editorial"
    }?,
```

Comment:
The `stream` attribute in `seriesInfo` in line 1099 needs to have `editorial` as an option too.

Reply:
Now added.